layouts/partials/head.html

```diff
@@ -29,6 +29,10 @@
   {{$scss := . | resources.ToCSS (dict "outputStyle" "compressed") }}
```

Comment:
this above range will include the `callouts.scss` all the time so the `if` check will need to happen inside the `range` on L28

Reply:
I cannot seem to get this working, do you have suggestion to get this to work?

Reply:
Basically, just prefix it with an `_` (so `_callouts.scss`) so it won't get matched by the resource regex above

Reply:
You know, that makes a lot of sense. I fixed it.